### PR TITLE
[Doc Improvement][Teams File Upload Bot][sbs guide][2124930]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3e15dbb05c3345ac3fd48059f50c2c150c977145/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5cc0497-18ea-47dd-b4f0-fa27d0f12967) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/65658dd0a9d3795b4c7043167e6af8c6f6b9dc16/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8278a66-f3e0-4a16-9e67-08c06ea9dcbd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/65658dd0a9d3795b4c7043167e6af8c6f6b9dc16/msteams-platform/sbs-file-handling-in-bot.yml) | 84 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbceed58-4fc0-4417-b491-a44046df86ac) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de0e48c30b97b985eb500f452ecda94eefda62c4/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1e8e2136-0361-4d51-8b19-1679d0b34069) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/de0e48c30b97b985eb500f452ecda94eefda62c4/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00174cd9-78f4-4fa2-9ccf-add0f1e0e372) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a4d0b75364ecb90c290961422498f8cab66efb19/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/316bff7a-bd9a-4fe8-bb6a-e0651599d985) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a4d0b75364ecb90c290961422498f8cab66efb19/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4e735a08-575b-4545-b5fb-17dc8b241c8d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/25a9565824453b1b81f8a0bde446e66cd9fa3537/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d71089d-f672-441b-81a1-7894be368e9e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/25a9565824453b1b81f8a0bde446e66cd9fa3537/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eaaab27b-037c-49f9-a72e-25fa61b4adca) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/25a9565824453b1b81f8a0bde446e66cd9fa3537/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/79f8d19d-680b-4fc0-8961-eeef4c98424c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aefa98a04e14d91021fffd025c1b6e26d939b297/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed1ec0b0-73f0-429a-bf3f-829264dc35ef) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/aefa98a04e14d91021fffd025c1b6e26d939b297/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ec4f803-8081-4b06-9675-db05b8c0ddc1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/aefa98a04e14d91021fffd025c1b6e26d939b297/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aec225e8-75dc-41c0-8c1a-5d832f33e336) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aefa98a04e14d91021fffd025c1b6e26d939b297/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/023df1ea-dafd-4b77-a784-3f1022c9d4c5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/34312222c501638b30d6cc6b9259af6a78e13c97/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28ab4b34-ce28-4c32-ac5e-5830c51d95b5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/34312222c501638b30d6cc6b9259af6a78e13c97/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5aa8ed26-f85b-4a47-8b3c-23cbeedaee61) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/34312222c501638b30d6cc6b9259af6a78e13c97/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e1c85ab8-7373-4c34-83b3-bbf446dcfe64) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/34312222c501638b30d6cc6b9259af6a78e13c97/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e580390-9427-4cca-a0ef-2d8b353b773b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea6d7c19b13d2efd6017796a1492033585c72eac/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b1a8163e-cc59-4bd1-9229-4ac263366c36) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea6d7c19b13d2efd6017796a1492033585c72eac/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d108cd8-95f2-4c68-9b86-c23855d1f1ed) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ea6d7c19b13d2efd6017796a1492033585c72eac/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3d658e32-8153-4039-9abf-a6de9b4b7b8e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ea6d7c19b13d2efd6017796a1492033585c72eac/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5892ec63-05c5-4f55-812e-bc4c89c3a24f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b5e47c978ece7edcd2df1c97aa51ff20668c01d5/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4048fea-4763-483d-93f0-e85c540968a7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b5e47c978ece7edcd2df1c97aa51ff20668c01d5/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2368ff5-e818-4b9e-9231-d8db439235cb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b5e47c978ece7edcd2df1c97aa51ff20668c01d5/msteams-platform/sbs-file-handling-in-bot.yml) | 83 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aed40f73-803d-4151-a498-bbf725e0d1e3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b5e47c978ece7edcd2df1c97aa51ff20668c01d5/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be69c0c3-210d-4780-b852-7f5414712c6c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/sbs-file-handling-in-bot.yml [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    1. Ensure that **State** is set to **Enabled**.
+
+        ![Add a scope](~/assets/images/teams-file-upload-bot/expo.png)
+
+    1. Select **Add scope**. The scope name automatically matches the Application ID URI, with **/access_as_user** added to the end.
+
+    **To authorize client applications:**
+
+    Add the following IDs as **Authorized client applications**, select the check box for **Authorized scopes** and select **Add application**.      
+    * 1fec8e78-bce4-4aaf-ab1b-5451cc387264 (Teams mobile or desktop application)
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-file-handling-in-bot.yml:204]: Are these the exact IDs to be used or will a user get different set of IDs. If this is representative, then obscure these or mention that this is an example and actual ID will be different.

@v-silakshmi

File: msteams-platform/sbs-file-handling-in-bot.yml [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+    1. Run your bot, from Visual Studio with `F5` or using `dotnet run` in the appropriate folder
+
+    1. Send a message `Hi` to the bot and wait for few seconds for it to respond with a card.
+
+- title: Interacting with the bot in Teams
+  durationInMinutes: 2
+  content: |
+    > [!NOTE]
+    >  The bot is installed in "personal" scope which is why you immediately enter a one on one chat conversation with the bot. See Teams documentation for more details.
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-file-handling-in-bot.yml:297]: If we suggest to see some docs, link to those.

@v-silakshmi

File: msteams-platform/sbs-file-handling-in-bot.yml [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+   :::image type="content" source="assets/images/teams-file-upload-bot/output-sbs.png" alt-text="sbs output" lightbox="assets/images/teams-file-upload-bot/output-sbs.png":::
+   
+- title: Prerequisites
+  durationInMinutes: 1 
+  content: |
+    Ensure that you install the following tools and set up your development environment:  
+
+    * Microsoft Teams is installed and you have an account
+    * [.NET Core SDK version 3.1](https://dotnet.microsoft.com/en-us/download)
```
Comment (ashishguptaiitb) [inline msteams-platform/sbs-file-handling-in-bot.yml:30]: Add URLs without en-us. Just check once that the links work without en-us language qualifier or not.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/74a4432ee32f69bbf1d904006e8e24ce6a7ee0f4/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac0a2b7d-80fd-41a4-a6fc-f59c32055a12) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/74a4432ee32f69bbf1d904006e8e24ce6a7ee0f4/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40d69649-0f5e-411f-bdaf-c952db63c47b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/74a4432ee32f69bbf1d904006e8e24ce6a7ee0f4/msteams-platform/sbs-file-handling-in-bot.yml) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d75ccac-7c98-40b4-ba93-1f8ceff9c1ff) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/74a4432ee32f69bbf1d904006e8e24ce6a7ee0f4/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/889ef436-7cde-444b-820e-fee65152f845) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/227de22f4f50b0782de9a47cb1c5ad88427a2158/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/416dc634-aed4-46ad-b25c-6a3fa3a3a050) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/227de22f4f50b0782de9a47cb1c5ad88427a2158/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a622382e-5062-40be-b806-534ba725df3c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/227de22f4f50b0782de9a47cb1c5ad88427a2158/msteams-platform/sbs-file-handling-in-bot.yml) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66ab5ef4-5719-4684-8eee-0af9980ffc89) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/227de22f4f50b0782de9a47cb1c5ad88427a2158/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6aaa2d91-7ce9-4548-a706-b27ff97a5dd5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-file-handling-in-bot.yml (hunk 3 above)
Comment (v-habibaumme) [reply msteams-platform/sbs-file-handling-in-bot.yml]: @ashishguptaiitb,

Sure, we'll check.

**Comment by ashishguptaiitb** [comment]
@v-silakshmi , please review my commit and fix/address anything that I got wrong. Once done, please confirm and I shall merge this PR.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2656ce4995f0b286986259cf3e423da44d225e2/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/606d97d7-6fc2-4a47-862b-e37745818549) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e2656ce4995f0b286986259cf3e423da44d225e2/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c7dd9b3-4774-4a44-afdf-5b7a548a6459) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e2656ce4995f0b286986259cf3e423da44d225e2/msteams-platform/sbs-file-handling-in-bot.yml) | 85 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd422d00-bd30-4022-818f-12bf6b0f99a2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e2656ce4995f0b286986259cf3e423da44d225e2/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d7379e45-e358-4470-923a-4da052fe80e2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bots-filesv4.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3804a0a076985651f202f0ba991abb0f500264a2/msteams-platform/bots/how-to/bots-filesv4.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2f7180c-72b8-452f-9cf1-e3a56f422c96) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3804a0a076985651f202f0ba991abb0f500264a2/msteams-platform/mstdd-landing.yml) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/60021525-a795-4ced-b6fd-fabe4253f78b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-file-handling-in-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3804a0a076985651f202f0ba991abb0f500264a2/msteams-platform/sbs-file-handling-in-bot.yml) | 86 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/752d75a4-74f4-439e-a446-0eb117608417) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3804a0a076985651f202f0ba991abb0f500264a2/msteams-platform/whats-new.md) | 92 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbcf249c-a67d-4021-95e0-3d4db40a0d9b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-file-handling-in-bot.yml (hunk 3 above)
Comment (v-silakshmi) [reply msteams-platform/sbs-file-handling-in-bot.yml]: have made changes as advised

File: msteams-platform/sbs-file-handling-in-bot.yml (hunk 2 above)
Comment (v-silakshmi) [reply msteams-platform/sbs-file-handling-in-bot.yml]: made changes to the document.

File: msteams-platform/sbs-file-handling-in-bot.yml (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/sbs-file-handling-in-bot.yml]: made changes as advised.

**Pull request merged**